HW1/P6/P6.py

```diff
@@ -0,0 +1,54 @@
+from pyspark import SparkContext
+import numpy as np
+from itertools import chain
+import random
+
+sc = SparkContext("local[4]", appName="HW1-6 Markov Shakespeare")
+sc.setLogLevel("ERROR")
+
+txt = open("Shakespeare.txt").readlines()
+txt = list(chain.from_iterable([line.split() for line in txt]))
```

Comment:
This is an interesting strategy, shifting the text at the input stage. Do you think there are pros and cons doing it here rather than, for example, after the words are stored in an RDD?